src/pybind/mgr/dashboard/frontend/src/app/shared/components/carbon-notification-panel/carbon-notification-panel.component.html

```diff
@@ -0,0 +1,89 @@
+<div class="cds--side-panel" [class.active]="isOpen">
+  <header class="cds--side-panel__header">
+    <h2 class="cds--side-panel__title">
+      <svg cdsIcon="notification" size="20"></svg>
+      Notifications
+    </h2>
+    <button class="cds--side-panel__close" (click)="close()">
+      <svg cdsIcon="close" size="20"></svg>
+    </button>
+  </header>
+
+  <div class="cds--side-panel__body">
+    <div class="cds--side-panel__controls">
+      <ibm-toggle
+        [checked]="doNotDisturb"
+        (change)="toggleDoNotDisturb()"
+        label="Do not disturb">
+      </ibm-toggle>
+
+      <button
+        cdsButton="ghost"
+        size="sm"
+        class="dismiss-all"
+        (click)="dismissAll()">
+        <svg cdsIcon="trash-can" size="20" class="cds--btn__icon"></svg>
+        Dismiss all
+      </button>
+    </div>
+
+    <div class="cds--side-panel__content">
+      <!-- Executing Tasks -->
+      <div class="executing-tasks" *ngIf="executingTasks.length > 0">
+        <h3 class="section-title">Running Tasks</h3>
```

Comment:
There is  header [component](https://github.com/carbon-design-system/carbon-components-angular/blob/v5.56.2/src/ui-shell/header/header.component.ts) what about that ?